components/automate-ui/src/app/entities/policies/policy.selectors.ts

```diff
@@ -16,9 +16,10 @@ export const iamMajorVersion = createSelector(
   (state) => state.iamMajorVersion
 );
 
+// This is used early enough that we have to guard against undefined (at least for unit tests)
 export const isIAMv2 = createSelector(
   policyState,
-  (state) => (state.iamMajorVersion === 'v2')
+  (state) => (state && state.iamMajorVersion === 'v2')
```

Comment:
The fixes above somehow caused this to fail a couple unit tests with state being undefined here.